components/txn_types/src/write.rs

```diff
@@ -68,6 +70,47 @@ pub struct Write {
     /// Also note that `has_overlapped_rollback` field is only necessary when the Rollback record
     /// should be protected.
     pub has_overlapped_rollback: bool,
```

Comment:
Seems it can be replaced by `gc_fence`?

Reply:
If it's produced because commit_ts == rollback_ts in the lock, it doesn't need a gc fence.